### ISSUE TITLE
Materials

### ISSUE THREAD

**Issue by jvcasillas** [jvcasillas/many_analyses#1]
- [ ] examine materials of previous studies and make list of what we need (i.e., specific surveys)
- [ ] update this list with specific materials once we have identified them

**Comment by stefanocoretta** [comment]
I started a draft of materials. Of course, feel free to move/edit/delete etc.

**Comment by jvcasillas** [comment]
Updated first comment to link to the materials draft.

**Comment by stefanocoretta** [comment]
Do we know if the Many EcoEvo materials are released under a particular licence? I am looking at the intake form and would be goof if I could keep most of it and just make edits (since a few things wouldn't change much)...

**Comment by troettge** [comment]
I can't see any assigned license but knowing Hannah, everything is public domain (or intended for public domain).

**Comment by troettge** [comment]
Guys, i have drafted a video to inform possible customers of the study. Its flashy, its simplified, and its narrated by a guy with a heavy German accent. Please have a look [here](https://osf.io/azdyw/). You have to download it for now. But it looks as if OSF as an in-build video player, so just need to find out which format it supports.

Tasks for you: 

- [ ] watch it ^^
- [ ] make notes on stuff you think needs changing
- [ ] we need a better concrete formulation of the research question. Its ugly as hell. Please make suggestions.
- [ ] discuss your notes next time we meet

**Comment by jvcasillas** [comment]
I really like it! The concept is perfect. It is clear and understandable. I am going to have my wife watch it too. I think we just need a clean version, possibly with better sound quality (mic)?

- At 1:48; "we are over informative here" (maybe consider "we would be...")

**Comment by troettge** [comment]
Yes absolutely: Better sound quality and a native speaker doing the voice over.

**Comment by troettge** [comment]
@jvcasillas @stefanocoretta I uploaded a trimmed version of the video (<4mins), leaving out a lot of background. I like that it's short but I also don't think the research questions is introduced very organically. I prefer the longer version. See what you think (in OSF repo)

**Comment by troettge** [comment]
updated version of the video up on OSF

**Comment by stefanocoretta** [comment]
> updated version of the video up on OSF

Looks/sounds good!!! I would only just silence the last bit of the audio, because you can hear the mouse click. Other than that, it's great!

**Comment by stefanocoretta** [comment]
I have added an `osf` folder with files for managing the OSF repo using osfr (bbfc10b, on `osfr` branch). I have also merged these changes into the `scdev` branch.

**Comment by troettge** [comment]
I looked at surveys and changed their look a little (added placeholder banner with logo). I edited a few questions and noted down some questions. I also have a couple of requests: @jvcasillas 

- we should add a description at the beginning of every form to help people navigate where they are in the process.
- set everything as required except the comment ones.
- The question after career stage: ERC is a little vague. Maybe more concrete or merge with postdoc?
- In the analytical survey, we should maybe give an example for each question
- the intake form, recruitment form, and reviewer form all have questions regarding name etc. Maybe we can condense these. Also the formulation of these questions are not the same, maybe streamline.
- The reviewer survey asks for ID code of analysis, maybe we can use the animal coding that Stefano suggested.
- I was wondering whether we still want to ask for the familiarity with stats methods. If so we probably should also ask for familiarity with acoustic measures? Or maybe we don't need it anymore.
